### PR TITLE
Updated to the latest Bacting

### PR DESCRIPTION
**Pull request by egonw**

### CODE REVIEW

**Comment by egonw** [comment]
It seems to fail on not having the required Java version? I am trying to figure out how to require Java 17.

**Comment by egonw** [comment]
Okay, the Java isse seems to be solved, but it seems I'm now running into a Sphinx bug: https://github.com/sphinx-doc/sphinx/issues/11678 :(

File: tox.ini [hunk 1]
```diff
@@ -121,6 +122,7 @@ description = Run the doc8 tool to check the style of the RST files in the proje
 
 [testenv:docs]
 changedir = docs
+passend = JAVA_HOME
```
Comment (cthoyt) [inline tox.ini:125]:
```suggestion
passenv = JAVA_HOME
```

Comment (egonw) [reply tox.ini:125]: duh

**Comment by egonw** [comment]
Update, I still haven't found what I was looking for, and still unsure how to force installing Sphinx 7.3 :/

**Comment by cthoyt** [comment]
@egonw is sphinx 7.3 a pre-release?

**Comment by egonw** [comment]
> @egonw is sphinx 7.3 a pre-release?

Yeah, not ideal, but that fixes the bug we run into: https://github.com/sphinx-doc/sphinx/issues/11678 (and the 7.3 changelog that also links to it)

**Review by cthoyt — APPROVED** [review]
Thanks @egonw. I've removed the sphinx coverage output, as there's already an alternative linting check that accomplishes the same thing. Will get this updated on PyPI momentarily :)

**Pull request merged**